main.py

```diff
@@ -1,10 +1,59 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as file:
+        players = json.load(file)
+
+    for player_name, player_data in players.items():
+        player_email = player_data["email"]
+        player_bio = player_data["bio"]
+
+        race_data = player_data["race"]
+
+        race_name = race_data["name"]
+        race_description = race_data["description"]
+
+        if not Race.objects.filter(name=race_name).exists():
+            race_obj = Race.objects.create(
+                name=race_name,
+                description=race_description
+            )
+        else:
+            race_obj = Race.objects.get(name=race_name)
```

Comment:
You can use get_or_create
https://docs.djangoproject.com/en/4.2/ref/models/querysets/#get-or-create
Fix in all places. Instead of use create + get + exists you can just use get_or_create